src/features/cookies/useCookiesChoiceByCategory.test.ts

```diff
@@ -0,0 +1,49 @@
+import { ALL_OPTIONAL_COOKIES, COOKIES_BY_CATEGORY } from 'features/cookies/CookiesPolicy'
+import { useCookiesChoiceByCategory } from 'features/cookies/useCookiesChoiceByCategory'
+
+describe('useCookiesChoiceByCategory', () => {
+  it('should past all cookies choice by category to false when refused all', () => {
```

Comment:
je suis mauvais en anglais, j'ai l'impression que ce n'est pas de l'anglais correct

---

d'un point de vue PO / UX / user, iels ne parlent généralement pas de `false`, j'essaie de nommer les règles qui sont vérifiées par les tests en réutilisant au maximum le langage utilisé par les PO / UX / users

que penserais-tu de ?

```suggestion
  it('should have every categories disabled when refusing all cookies', () => {
```

Reply:
Oui tu as raison, je n'arrivais pas a trouver le bon wording :D